### PR TITLE
UIFC-226-tests-for-contacts add tests for edit and view

### PR DESCRIPTION
**Pull request by elsenhans**

### CODE REVIEW

File: test/bigtest/network/factories/finc-config-metadata-source.js [hunk 1]
```diff
@@ -11,7 +12,7 @@ export default Factory.extend({
     id: '',
     name: ''
   },
-  contacts: [],
+  contacts : [],
```
Comment (rchr) [inline test/bigtest/network/factories/finc-config-metadata-source.js:15]: Can you please delete the whitespace between 'contacts' and ':'?

File: test/bigtest/network/scenarios/default.js [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
 // default scenario is used during `yarn start --mirage`
 export default function defaultScenario(server) {
-  server.create('finc-config-metadata-source');
+  // server.create('finc-config-metadata-source');
```
Comment (rchr) [inline test/bigtest/network/scenarios/default.js:5]: This line is not used anymore. It is save to delete it.

**Review by rchr — APPROVED** [review]
Great work! Thank you!

There are just two minor comments...

**Comment by elsenhans** [comment]
Thanks @rchr!

**Pull request merged**